### ISSUE TITLE
Invalid regular expression error

### ISSUE THREAD

**Issue by macwish** [ivanakimov/hashids.node.js#26]
## Versions
hashids-node: 1.0.2
Node: 5.1.1
OS: Mac OS 10.11.5

--------

```Javascript
'use strict';

//
// File name: Untitled.js
// 
// npm install hashids
// node Untitled.js
//

const crypto = require('crypto');
const Hashids = require("hashids");

const NUMBER = '0123456789';
const APHAABET = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
const CHARS = '!#*+-:=?';	
// const CHARS = '!#*+-:=';  // but, this line works well.

const sets = NUMBER + APHAABET + CHARS;
const hashids = new Hashids('sssse', 0, sets);
// SyntaxError: Invalid regular expression

var num = 1234;
var id = hashids.encode(num);
console.log('raw: %d', num);
console.log('enc: %s', id);
console.log('dec: ' + hashids.decode(id));
```

-----
Output:
```
/Users/apple/hashids-test/node_modules/hashids/lib/hashids.js:254
		r = new RegExp("[" + this.guards + "]", "g"),
		    ^

SyntaxError: Invalid regular expression: /[ZA-*O]/: Range out of order in character class
	at new RegExp (native)
	at Hashids._decode (/Users/apple/hashids-test/node_modules/hashids/lib/hashids.js:254:7)
	at Hashids.decode (/Users/apple/hashids-test/node_modules/hashids/lib/hashids.js:147:14)
	at Object.<anonymous> (/Users/apple/hashids-test/Untitled3.js:29:21)
	at Module._compile (module.js:425:26)
	at Object.Module._extensions..js (module.js:432:10)
	at Module.load (module.js:356:32)
	at Function.Module._load (module.js:313:12)
	at Function.Module.runMain (module.js:457:10)
	at startup (node.js:138:18)
```

@ivanakimov

**Comment by ivanakimov** [comment]
Thanks for the bug report @macwish. Two things:

1. This has been fixed in [1.1.0](https://github.com/ivanakimov/hashids.js/blob/master/CHANGELOG.md)
2. This repo has moved to <https://github.com/ivanakimov/hashids.js>

**Comment by macwish** [comment]
Well done! 👍  @ivanakimov
